decidim-assemblies/app/controllers/decidim/assemblies/assemblies_controller.rb

```diff
@@ -87,6 +87,10 @@ def stats
       def assembly_participatory_processes
         @assembly_participatory_processes ||= @current_participatory_space.linked_participatory_space_resources(:participatory_processes, "included_participatory_processes")
       end
+
+      def current_assemblies_settings
+        @current_assemblies_settings = Decidim::AssembliesSetting.find_or_create_by(decidim_organization_id: current_organization.id)
```

Comment:
As commented in the other controller, I'd suggest to memoize this one also:

```suggestion
        @current_assemblies_settings ||= Decidim::AssembliesSetting.find_or_create_by(decidim_organization_id: current_organization.id)
```